requirements.txt

```diff
@@ -601,6 +601,10 @@ pyyaml==6.0 \
     # via
     #   -r requirements.in
     #   drf-spectacular
+q==2.7 \
```

Comment:
What is this used for?

Reply:
For debugging. I'll remove it before submitting a proper PR for the final review.